include/smack/SmackRep.h

```diff
@@ -150,6 +166,14 @@ class SmackRep {
   virtual const Stmt* alloca(llvm::AllocaInst& i);
   virtual const Stmt* memcpy(const llvm::MemCpyInst& msi);
   virtual const Stmt* memset(const llvm::MemSetInst& msi);
+  virtual const Stmt* loadAsBytes(const llvm::LoadInst& li);
+  virtual const Stmt* storeAsBytes(const llvm::StoreInst& si);
```

Comment:
In load/store methods, we probably need a branch like,

`if (...(SmackOptions::InferFieldOverlap && rep.isFieldDisjoint(..)))`

`SmackRep::isFieldDisjoint()` requires an argument of class `llvm::Function`. My original doubt is whether this argument could be derived from the two arguments of `store(const llvm::Value* addr, const llvm::Value* val)`. Now it seems it could.